src/test/resources/expected-classes/extend-with-resolvers/Node.java.txt

```diff
@@ -2,9 +2,11 @@
     value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
     date = "2020-12-31T23:59:59-0500"
 )
+@com.fasterxml.jackson.annotation.JsonTypeInfo(use = com.fasterxml.jackson.annotation.JsonTypeInfo.Id.NAME, property = "__typename")
+@com.fasterxml.jackson.databind.annotation.JsonTypeIdResolver(GraphqlJacksonTypeIdResolver.class)
```

Comment:
I think we should add a fully qualified package name here for `GraphqlJacksonTypeIdResolver`